test/test-util/schema.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Mirrored types from router-api for use in tests
+ */
+
+import { MethodParameters } from '@uniswap/v3-sdk';
+
+export type TokenInRoute = {
+  address: string;
+  chainId: number;
+  symbol: string;
+  decimals: string;
+};
+
+export type BasicPoolInRoute = {
+  type: string;
+  amountIn?: string;
+  amountOut?: string;
+};
+
+export type V3PoolInRoute = {
+  type: 'v3-pool';
+  address: string;
+  tokenIn: TokenInRoute;
+  tokenOut: TokenInRoute;
+  sqrtRatioX96: string;
+  liquidity: string;
+  tickCurrent: string;
+  fee: string;
+  amountIn?: string;
+  amountOut?: string;
+};
+
+export type V2PoolInRoute = {
+  type: 'v2-pool';
+  address: string;
+  tokenIn: TokenInRoute;
+  tokenOut: TokenInRoute;
+  reserve0: V2Reserve;
+  reserve1: V2Reserve;
+  amountIn?: string;
+  amountOut?: string;
+};
+
+export type V2Reserve = {
+  token: TokenInRoute;
+  quotient: string;
+};
+
+export type QuoteResponse = {
```

Comment:
This type is no longer used (in favor of more concise `

Reply:
hmm...as a reference to how the `routing-api` works? Unless there's a comment, I'd be afraid someone would come along and see this as dead code and delete. Not fully convinced it should be included.

Reply:
ok I see..maybe you could look into typescript utility types to keep it around? using a partial or maybe omit https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
My initial rationale was to keep it both as reference to the routing api and to show what fields can be extracted from the swap response. I'm leaning towards adding a comment marking it is deprecated

Reply:
Could leave a comment. but.. QuoteResponse in the API is not deprecated.. You just mean you're not using it in the tests right?

Reply:
yea its not deprecated, just not being used. Can mark as unsued

Reply:
now thinking about it, if we're importing more things from SOR into routing-api, we may likely end up importing this type to use as well.